doc/release/1.1.0-notes.rst

```diff
@@ -29,7 +29,7 @@ New features
 ------------------------------
 
 The argument ``tfirst`` has been added to the function
-`scipy.integrate.odeint`. This allows odeint to use the same user
+`scipy.integrate.odeint`. This allows ``odeint'' to use the same user
```

Comment:
Note you added backticks at the beginning but regular quotes at the end of `odeint` - this will break sphinx

Reply:
Just figured out that commas alone couldn't have broken it down. Thanks for pointing this out!